website/sidebars.js

```diff
@@ -331,16 +331,8 @@ const sidebarSettings = {
       items: [
         "docs/deploy/job-scheduler",
         "docs/deploy/deploy-environments",
-        {
-          type: "category",
-          label: "dbt Cloud jobs",
-          link: { type: "doc", id: "docs/deploy/dbt-cloud-job" },
-          items: [
-            "docs/deploy/job-settings",
-            "docs/deploy/job-commands",
-            "docs/deploy/job-triggers",
-          ],
-        },
+        "docs/deploy/jobs",
```

Comment:
wondering if it makes sense make Deploy jobs and CI as subpages under this main Jobs page. wdyt, @schottj ?